tests/tabtie_tests.ts

```diff
@@ -33,7 +33,7 @@ const TabTieTests = {
 /**
  * Helper function to create TabNote objects.
  */
-const tabNote = (struct: TabNoteStruct) => new TabNote(struct);
+const tabNote = (noteStruct: Partial<TabNoteStruct>) => new TabNote(noteStruct);
```

Comment:
This is what I mean about "slightly more keystrokes or mental overhead" for the developer.

If someone wants to create a helper method, they might have to type out `Partial<TabNoteStruct>` because using just `TabNoteStruct` here will trigger a TS compiler error.

Reply:
We have to use Partial due to the fact that we decided to use Partial definitions in the tests. If we defined the complete structure in the test Partial would not be required.